bin/ch/WScriptJsrt.cpp

```diff
@@ -947,18 +947,16 @@ bool WScriptJsrt::Initialize()
         IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "Leaving", LeavingCallback));
         IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "Sleep", SleepCallback));
 
-
-        // OSX does build does not support $262 as filename
-        const wchar_t $262[] =
+        const char Test262[] =
```

Comment:
While we are changing code here, can we add a comment here and/or in 262.js that explains this code pattern?

Reply:
Good idea.